app/layout.js

```diff
@@ -6,18 +6,35 @@ import "swiper/css/pagination";
 import "swiper/css/autoplay";
 import "swiper/css/navigation";
 import FloatingBtn from "./components/Floating-btn-component/FloatingBtn";
+import Head from "next/head";
+import { Metadata } from 'next';
+
+export const metadata = {
+  title: 'PTGA',
+  description: 'Cooperativa especializada en asesoramiento, investigación, fabricación y serviciosde instalación para el desarrollo de tecnología en telecomunicaciones',
+
+};
+
 
 export default function RootLayout({ children }) {
   return (
-    <html lang="en">
+
+    <html lang="es">
+      <Head>
```

Comment:
utilizar el componente head tira un warning

Warning: You're using `next/head` inside the `app` directory, please migrate to the Metadata API. See https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#step-3-migrating-nexthead for more details.

Reply:
En este momento de revision y edicion apartir de comentarios no registro errores o warnings. Podremos checkearlo juntos?